<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>QShodan</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_web">
       <attribute name="title">
        <string>Buscar</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btn_home">
            <property name="text">
             <string>Home</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_extract">
            <property name="text">
             <string>Extraer IPs de esta pagina</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_autoextract">
            <property name="text">
             <string>AutoExtraer</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWebView" name="webView">
          <property name="url">
           <url>
            <string>https://www.shodan.io/</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_export">
       <attribute name="title">
        <string>Exportar</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListWidget" name="list_export">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="table_export"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="btn_clean">
            <property name="text">
             <string>Limpiar Lista</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_duplicates">
            <property name="text">
             <string>Borrar Duplicados</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_save_csv">
            <property name="text">
             <string>Save CSV...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btn_clean</sender>
   <signal>released()</signal>
   <receiver>table_export</receiver>
   <slot>clearContents()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>733</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clean</sender>
   <signal>released()</signal>
   <receiver>list_export</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>778</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
